### PR TITLE
feat: update table name and condition for recent 24h

### PR DESCRIPTION
**Pull request by jhlee-young**

### CODE REVIEW

File: api/service/stat.go [hunk 1]
```diff
@@ -40,7 +40,8 @@ func (s *statService) Get(key string) (*PairStats, error) {
 
 	switch key {
 	case statPeriod24h:
-		err := s.sumPairStats24h(0, pairStatMap)
+		minTs := time.Now().Add(-24 * time.Hour)
```
Comment (jbamlee) [inline api/service/stat.go:43]: nit.
```suggestion
		minTs := time.Now().Sub(24 * time.Hour)
```

File: api/service/stat.go [hunk 2]
```diff
@@ -119,25 +120,28 @@ func (s *statService) GetAll() ([]PairStats, error) {
 }
 
 func (s *statService) sumPairStats24h(minTimestamp float64, sumStatMap map[string][countOfStatType]types.Dec) error {
```
Comment (jbamlee) [inline api/service/stat.go:122]: nit. clear naming 
```suggestion
func (s *statService) sumPairStatsFrom (fromTs float64, sumStatMap map[string][countOfStatType]types.Dec) error {
```

**Review by jbamlee — APPROVED** [review]

File: api/service/stat.go (hunk 1 above)
Comment (jhlee-young) [reply api/service/stat.go:43]: `Sub` needs `time.Time`, not `time.Duration`

File: api/service/stat.go (hunk 2 above)
Comment (jhlee-young) [reply api/service/stat.go]: [517e9a6](https://github.com/dezswap/dezswap-api/pull/10/commits/517e9a62be905c4b84179d6c15852bed7114ea1e)

**Pull request merged**